domino_generator/example/ex1/ex1.g.dart

```diff
@@ -37,12 +39,18 @@ void renderEx1(
   }
   $d.close();
   if (extra) {
-    renderEx1($d, obj: obj, extra: false);
+    $dSlots[''] = (_i0.DomContext $d) {};
+    Ex1($d, obj: obj, extra: false, $dSlots: $dSlots);
```

Comment:
Calling an object with slots should be local to the component, do not override the render-method's `$dSlots` parameter. If the current component has internal slots initialize them locally, e.g.:

````
renderEx1($d, obj: obj, extra: false, slotA: (_i0.DomContext $d) {});
````